src/main/js/main_html5.js

```diff
@@ -888,18 +888,22 @@ require("../../../html5-common/js/utils/environment.js");
         var currentAudio = _.find(audioTracks, function (track) {
           return track.enabled;
         });
-        var currentAudioId = currentAudio.id;
-        if (currentAudioId !== trackId) {
-          var newAudioTrack = audioTracks.getTrackById(trackId);
-          if (newAudioTrack) { // if trackId is correct and the audio exists
-            var prevAudioTrack = audioTracks.getTrackById(currentAudioId);
-            if (prevAudioTrack) { // if currentAudioId is correct and the audio exists
-              prevAudioTrack.enabled = false; // the audio is not active anymore
+        var currentAudioId = null;
+        if (currentAudio && currentAudio.id) {
+          currentAudioId = currentAudio.id;
+          if (currentAudioId !== trackId) {
+            var newAudioTrack = audioTracks.getTrackById(trackId);
+            if (newAudioTrack) { // if trackId is correct and the audio exists
+              var prevAudioTrack = audioTracks.getTrackById(currentAudioId);
+              if (prevAudioTrack) { // if currentAudioId is correct and the audio exists
+                prevAudioTrack.enabled = false; // the audio is not active anymore
+              }
+              newAudioTrack.enabled = true; // the audio is active
```

Comment:
Will this get set if the currentAudio is null? I believe is there is no current audio, we should still set the new audio track.

Reply:
Thanks for the review! 
I think right now we handle only those tracks that were added synchronously, and by design  so there'll be no situation where there's no track being active at some point.

Reply:
Sounds good.